apps/agreement/test/agreement/agreement_lock_manager.js

```diff
@@ -0,0 +1,82 @@
+const deployer = require('../helpers/utils/deployer')(web3, artifacts)
+const { injectWeb3, injectArtifacts } = require('@aragon/contract-helpers-test')
+const { RULINGS } = require('../helpers/utils/enums')
+
+injectWeb3(web3)
+injectArtifacts(artifacts)
+
+contract('Agreement', ([_, owner, submitter, challenger, someone]) => {
+  let disputable, actionCollateral, collateralToken
+  let agreement
+
+  before('deploy agreement instance', async () => {
+    disputable = await deployer.deployAndInitializeDisputableWrapper({ owner, activate: false, submitters: [submitter], challengers: [challenger] })
+    agreement = disputable.agreement
+    collateralToken = disputable.collateralToken
+    actionCollateral = disputable.actionCollateral
+  })
+
+  describe('lock manager interface', () => {
+    let actionId
+
+    context('can’t unlock', () => {
```

Comment:
```suggestion
    context("can't unlock", () => {
```

Might be a bit easier 😄

Reply:
hm, ok, but ask @bpierre about it. He convinced me to use ’, “ and ” a while ago :wink: